class USBError(Exception):
    def __init__(self) -> None:
        pass
